contracts/scripts/validator-registry/DeployValidatorOptInHub.s.sol

```diff
@@ -8,89 +8,67 @@ pragma solidity 0.8.26;
 import {Script} from "forge-std/Script.sol";
```

Comment:
Imo we should create an entirely new deployment script since we are introducing a new contract here. Separately we can create another file for updating the old registry to ref the new one for backwards compatibility